### PR TITLE
{devel,tools,vis}[GCCcore/6.3.0] GLib v2.53.5, PCRE v8.41, util-linux v2.30.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires #5036

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2c64b4f9d6e773bee688384f5bcc42a8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c2e62aa74ccd291a078f3ae6869f6958 for a full test report.

File: easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+configopts = "--disable-maintainer-mode --disable-silent-rules --disable-libelf --disable-systemtap "
+configopts += "--enable-static --enable-shared"
+
+postinstallcmds = ["sed -i -e 's|#!.*python|#!/usr/bin/env python|' %(installdir)s/bin/*"]
+
+sanity_check_paths = {
+    'files': ['lib/libglib-%(version_major)s.0.a', 'lib/libglib-%%(version_major)s.0.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include'],
+}
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb]: @boegel Why did you remove the `modextrapaths`?

File: easybuild/easyconfigs/u/util-linux/util-linux-2.30.1-GCCcore-6.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+description = "Set of Linux utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['%s/v%%(version_major_minor)s' % homepage]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['88bb1d08f0d67fb1b548c3e2160f7f40fb552cdf6b771a5dd1b94e1e04fc1d9a']
+
+# disable account related utilities (they need OS dependant pam-devel files)
```
Comment (verdurin) [inline easybuild/easyconfigs/u/util-linux/util-linux-2.30.1-GCCcore-6.3.0.eb]: @boegel `dependent`

File: easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb]:
```python
modextrapaths = {'CPATH': ['include/glib-2.0', 'lib/glib-2.0/include']}
```

Hmm, this is only present in one single (fairly old) `GLib` easyconfig (`GLib-2.44.0-GCC-4.9.2.eb`).

It seems like this is not needed at all in the others, which may mean it's not required there either...

Should be easy to tell, since it's only used as a dep for `Qt-4.8.6-foss-2015a-GLib-2.44.0.eb` (which in turns is a dep for `CGAL-4.6-*-2015a-Python-2.7.*.eb`), nowhere else...

Comment (verdurin) [inline easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.3.0.eb]: Fair enough.

**Comment by verdurin** [comment]
Going in, thanks @boegel!
